### ISSUE TITLE
[Bug] LUA Error "iconPicker" when open Weakauras (your ptched Version if this matters)

### ISSUE THREAD

**Issue by fubaWoW** [fang2hou/ElvUI_WindTools#395]
**Describe the bug**
LUA Error when open Weakauras (your patched Version if this matters, but i think not)

**To Reproduce**
1. Open Weakauras with ``/wa``
2. See error

**Expected behavior**
no error ^^

**Screenshots**
none

**Information (you can find these in [WindTools-Information]):**
 - Addon Version: 3.58
 - ElvUI Version 13.69
 - WoW Build 10.2.7 (55461)

**Additional context**
Lua Error here:
```LUA
3x ..._WindTools/Modules/Skins/Addons/WeakAurasOptions.lua:672: attempt to index field 'iconPicker' (a nil value)
[string "@ElvUI_WindTools/Modules/Skins/Addons/WeakAurasOptions.lua"]:672: in function <..._WindTools/Modules/Skins/Addons/WeakAurasOptions.lua:478>
[string "=(tail call)"]: ?
[string "=[C]"]: in function `ShowOptions'
[string "@WeakAurasOptions/WeakAurasOptions.lua"]:612: in function `ToggleOptions'
[string "@WeakAuras/WeakAuras.lua"]:171: in function `OpenOptions'
[string "@WeakAuras/WeakAuras.lua"]:253: in function `?'
[string "@Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5422: in function <Blizzard_ChatFrameBase/Mainline/ChatFrame.lua:5368>
[string "=[C]"]: ?
[string "=[C]"]: in function `ChatEdit_ParseText'
[string "@Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5074: in function <Blizzard_ChatFrameBase/Mainline/ChatFrame.lua:5073>
[string "=[C]"]: in function `ChatEdit_SendText'
[string "@Blizzard_ChatFrameBase/Mainline/ChatFrame.lua"]:5110: in function <Blizzard_ChatFrameBase/Mainline/ChatFrame.lua:5106>
[string "=[C]"]: in function `ChatEdit_OnEnterPressed'
[string "*ChatFrame.xml:140_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:140_OnEnterPressed"]:1>

Locals:
self = <table> {
 CreateLowerShadow = <function> defined @ElvUI_WindTools/Modules/Skins/Core.lua:100
 TalkingHead = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/TalkingHead.lua:6
 UpdateShadowColor = <function> defined @ElvUI_WindTools/Modules/Skins/Core.lua:127
 HandleMerchantItem = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/Merchant.lua:8
 ElvUI_AFK = <function> defined @ElvUI_WindTools/Modules/Skins/ElvUI/AFK.lua:7
 modules = <table> {
 }
 CancelTimer = <function> defined @WeakAuras/Libs/AceTimer-3.0/AceTimer-3.0.lua:147
 GuildInviteFrame = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/Guild.lua:6
 MirrorTimers = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/MirrorTimers.lua:7
 RaiderIO = <function> defined @ElvUI_WindTools/Modules/Skins/Addons/RaiderIO.lua:137
 SkinCriteriaAlert = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/Alerts.lua:63
 WeakAuras_PrintProfile = <function> defined @ElvUI_WindTools/Modules/Skins/Addons/WeakAuras.lua:14
 Blizzard_LossOfControlFrame_SetUpDisplay = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/LossOfControl.lua:6
 ElvUI_TotemTracker_Initialize = <function> defined @ElvUI_WindTools/Modules/Skins/ElvUI/TotemTracker.lua:8
 Blizzard_WeeklyRewards = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/WeeklyRewards.lua:6
 OmniCD_Party_ExtraBars = <function> defined @ElvUI_WindTools/Modules/Skins/Addons/OmniCD.lua:40
 Blizzard_PlayerChoice = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/PlayerChoice.lua:29
 SkinItemButton = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/ObjectiveTracker.lua:24
 UpdateFriendButton = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/Friends.lua:9
 ADDON_LOADED = <function> defined @ElvUI_WindTools/Modules/Skins/Core.lua:354
 WeakAurasIconButton = <function> defined @ElvUI_WindTools/Modules/Skins/Addons/WeakAurasOptions.lua:436
 ElvUI_CastBars = <function> defined @ElvUI_WindTools/Modules/Skins/ElvUI/CastBars.lua:59
 SettingsPanel = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/SettingsPanel.lua:8
 TutorialFrame = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/Tutorial.lua:6
 ElvUI_TotemTracker = <function> defined @ElvUI_WindTools/Modules/Skins/ElvUI/TotemTracker.lua:18
 SetEnabledState = <function> defined @ArkInventory/Externals/Ace3/AceAddon/AceAddon-3.0.lua:420
 Hook = <function> defined @ArkInventory/Externals/Ace3/AceHook/AceHook-3.0.lua:273
 enteredLoad = <table> {
 }
 DressUpFrame = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/DressingRoom.lua:6
 TomCats_HeaderCollapseButton_SetNormalAtlas = <function> defined @ElvUI_WindTools/Modules/Skins/Addons/TomCats.lua:61
 BigWigs_Plugins = <function> defined @ElvUI_WindTools/Modules/Skins/Addons/BigWigs.lua:264
 Blizzard_AnimaDiversionUI = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/AnimaDiversion.lua:6
 UnregisterAllEvents = <function> defined @AllTheThings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 Blizzard_ItemInteractionUI = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/ItemInteraction.lua:6
 Blizzard_ProfessionsCustomerOrders = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/ProfessionsCustomerOrders.lua:8
 ElvUI_UnitFrames_Construct_AuraBars = <function> defined @ElvUI_WindTools/Modules/Skins/ElvUI/UnitFrames.lua:84
 SkinWorldQuestCompleteAlert = <function> defined @ElvUI_WindTools/Modules/Skins/Blizzard/Alerts.lu
```

**Comment by fang2hou** [comment]
Fixed in 424f634bc7dd67861f0cdadd60ef0f5758439ac5.

Because the bug occured by WeakAuras update, I also update the WeakAurasPatched.
Re-install WeakAurasPatched with repatched 5.15.0 once is suggested.

**Comment by fubaWoW** [comment]
Thank you very much! 😁
